src/test/java/com/artipie/api/ManageUsersTest.java

```diff
@@ -162,6 +162,39 @@ void removesUser() {
         );
     }
 
+    @Test
+    void altersPassword() {
```

Comment:
Think we should have a test that checks the case when the old password is invalid.